### ISSUE TITLE
error compiling

### ISSUE THREAD

**Issue by AnimaI** [oscam-emu/oscam-emu#291]
`patching file CMakeLists.txt
patching file Makefile
patching file config.h
Hunk #1 succeeded at 1 with fuzz 1.
patching file config.sh
patching file cscrypt/aes.c
patching file cscrypt/aes.h
patching file cscrypt/md5.c
patching file cscrypt/md5.h
patching file csctapi/cardreaders.h
patching file ffdecsa/CMakeLists.txt
patching file ffdecsa/COPYING
patching file ffdecsa/Makefile
patching file ffdecsa/ffdecsa.c
patching file ffdecsa/ffdecsa.h
patching file ffdecsa/fftable.h
patching file ffdecsa/parallel_032_int.h
patching file ffdecsa/parallel_064_long.h
patching file ffdecsa/parallel_128_sse2.h
patching file ffdecsa/parallel_generic.h
patching file ffdecsa/parallel_std_def.h
patching file ffdecsa/stream.c
patching file globals.h
patching file module-dvbapi.c
patching file module-dvbapi.h
patching file module-emulator-biss.c
patching file module-emulator-biss.h
patching file module-emulator-cryptoworks.c
patching file module-emulator-cryptoworks.h
patching file module-emulator-director.c
patching file module-emulator-director.h
patching file module-emulator-drecrypt.c
patching file module-emulator-drecrypt.h
patching file module-emulator-irdeto.c
patching file module-emulator-irdeto.h
patching file module-emulator-nagravision.c
patching file module-emulator-nagravision.h
patching file module-emulator-osemu.c
patching file module-emulator-osemu.h
patching file module-emulator-powervu.c
patching file module-emulator-powervu.h
patching file module-emulator-streamserver.c
patching file module-emulator-streamserver.h
patching file module-emulator-viaccess.c
patching file module-emulator-viaccess.h
patching file module-emulator-videoguard.c
patching file module-emulator-videoguard.h
patching file module-emulator.c
patching file module-newcamd-des.c
patching file module-newcamd-des.h
patching file module-newcamd.c
patching file module-stat.c
patching file module-webif-tpl.c
patching file module-webif.c
patching file oscam-aes.c
patching file oscam-aes.h
patching file oscam-chk.c
patching file oscam-config-global.c
patching file oscam-config-reader.c
patching file oscam-config.h
patching file oscam-emm.c
patching file oscam-string.c
patching file oscam-string.h
patching file oscam.c
patching file reader-common.c
Hunk #1 succeeded at 142 (offset -1 lines).
patching file readers.h
patching file webif/config/dvbapi.html
patching file webif/config/menu.html
patching file webif/config/menu_streamrelay.html
patching file webif/config/streamrelay.html
patching file webif/files/menu.html
patching file webif/files/menu_softcamkey.html
patching file webif/pages_index.txt
patching file webif/readerconfig/readerconfig_emubit.html

+-------------------------------------------------------------------------------
| OSCam ver: 1.20_svn rev: 11438 target: x86_64-linux-gnu-ssl-libusb
| Tools:
|  CROSS    = 
|  CC       = gcc
| Settings:
|  CONF_DIR = /usr/local/etc
|  CC_OPTS  = -Os -ggdb -pipe -ffunction-sections -fdata-sections
|  CC_WARN  = -w
|  CFLAGS   = -fexpensive-optimizations -mmmx -msse -msse2 -msse3 -DWITH_SSL=1 -DWITH_LIBCRYPTO=1 -DWITH_LIBUSB=1
|  LDFLAGS  = -Wl,--gc-sections -DWITH_SSL=1 -DWITH_LIBCRYPTO=1 -DWITH_LIBUSB=1 -lz -Wl,-rpath,/usr/lib
|  LIBS     = -lssl -lcrypto -lusb-1.0 -lrt -lm -lpthread -ldl
|  UseFlags = USE_SSL=1 USE_LIBCRYPTO=1 USE_LIBUSB=1
| Config:
|  Addons   : WEBIF WEBIF_LIVELOG WITH_SSL WITH_DEBUG WITH_LB WITH_EMU
|  Protocols: CAMD35 CAMD35_TCP NEWCAMD CCCAM CCCSHARE
|  Readers  : NAGRA IRDETO CONAX CRYPTOWORKS SECA VIACCESS VIDEOGUARD DRE TONGFANG BULCRYPT GRIFFIN DGCRYPT
|  CardRdrs : SMARGO SMART
|  Compiler : gcc (Debian 4.9.2-10+deb8u1) 4.9.2
|  Config   : build/x86_64-linux-gnu-ssl-libusb/config.mak
|  Binary   : ../archiv/oscam-svn11438-oscam2
+-------------------------------------------------------------------------------
HOSTCC	webif/pages_gen
GEN	webif/pages.c
GEN	Compressed 441816 template bytes into 178931 bytes. 262885 saved bytes (59.50%).
CC	cscrypt/i_cbc.c
CC	cscrypt/des.c
CC	cscrypt/i_ecb.c
CC	cscrypt/i_skey.c
CC	cscrypt/rc6.c
CC	cscrypt/md5.c
CC	csctapi/atr.c
CC	csctapi/icc_async.c
CC	csctapi/io_serial.c
CC	csctapi/protocol_t0.c
CC	csctapi/protocol_t1.c
CC	csctapi/ifd_smargo.c
CC	csctapi/ifd_smartreader.c
CC	minilzo/minilzo.c
CC	module-camd35.c
CC	module-cccam.c
CC	module-cccshare.c
CC	module-emulator.c
CC	module-emulator-osemu.c
CC	module-emulator-streamserver.c
CC	module-emulator-biss.c
CC	module-emulator-cryptoworks.c
CC	module-emulator-director.c
CC	module-emulator-drecrypt.c
CC	module-emulator-irdeto.c
CC	module-emulator-nagravision.c
CC	module-emulator-powervu.c
CC	module-emulator-viaccess.c
CC	module-emulator-videoguard.c
CC	ffdecsa/ffdecsa.c
CC	module-newcamd.c
CC	module-newcamd-des.c
CC	module-stat.c
CC	module-webif-lib.c
CC	module-webif-tpl.c
CC	module-webif.c
CC	webif/pages.c
CC	reader-common.c
reader-common.c: In function ‘reader_get_cardsystem’:
reader-common.c:150:43: error: ‘reader_emu’ undeclared (first use in this function)
   rdr_log(reader, "found card system %s", reader_emu.desc);
                                           ^
reader-common.c:150:43: note: each undeclared identifier is reported only once for each function it appears in
Makefile:452: recipe for target 'build/x86_64-linux-gnu-ssl-libusb/reader-common.o' failed
make[1]: *** [build/x86_64-linux-gnu-ssl-libusb/reader-common.o] Error 1
make[1]: *** Waiting for unfinished jobs....
Makefile:404: recipe for target 'all' failed
make: *** [all] Error 2

entferne Patch(e) oscam-emu.patch
patching file CMakeLists.txt
patching file Makefile
patching file config.h
Hunk #1 succeeded at 1 with fuzz 1.
patching file config.sh
patching file cscrypt/aes.c
patching file cscrypt/aes.h
patching file cscrypt/md5.c
patching file cscrypt/md5.h
patching file csctapi/cardreaders.h
patching file ffdecsa/CMakeLists.txt
patching file ffdecsa/COPYING
patching file ffdecsa/Makefile
patching file ffdecsa/ffdecsa.c
patching file ffdecsa/ffdecsa.h
patching file ffdecsa/fftable.h
patching file ffdecsa/parallel_032_int.h
patching file ffdecsa/parallel_064_long.h
patching file ffdecsa/parallel_128_sse2.h
patching file ffdecsa/parallel_generic.h
patching file ffdecsa/parallel_std_def.h
patching file ffdecsa/stream.c
patching file globals.h
patching file module-dvbapi.c
patching file module-dvbapi.h
patching file module-emulator-biss.c
patching file module-emulator-biss.h
patching file module-emulator-cryptoworks.c
patching file module-emulator-cryptoworks.h
patching file module-emulator-director.c
patching file module-emulator-director.h
patching file module-emulator-drecrypt.c
patching file module-emulator-drecrypt.h
patching file module-emulator-irdeto.c
patching file module-emulator-irdeto.h
patching file module-emulator-nagravision.c
patching file module-emulator-nagravision.h
patching file module-emulator-osemu.c
patching file module-emulator-osemu.h
patching file module-emulator-powervu.c
patching file module-emulator-powervu.h
patching file module-emulator-streamserver.c
patching file module-emulator-streamserver.h
patching file module-emulator-viaccess.c
patching file module-emulator-viaccess.h
patching file module-emulator-videoguard.c
patching file module-emulator-videoguard.h
patching file module-emulator.c
patching file module-newcamd-des.c
patching file module-newcamd-des.h
patching file module-newcamd.c
patching file module-stat.c
patching file module-webif-tpl.c
patching file module-webif.c
patching file oscam-aes.c
patching file oscam-aes.h
patching file oscam-chk.c
patching file oscam-config-global.c
patching file oscam-config-reader.c
patching file oscam-config.h
patching file oscam-emm.c
patching file oscam-string.c
patching file oscam-string.h
patching file oscam.c
patching file reader-common.c
Hunk #1 succeeded at 142 (offset -1 lines).
patching file readers.h
patching file webif/config/dvbapi.html
patching file webif/config/menu.html
patching file webif/config/menu_streamrelay.html
patching file webif/config/streamrelay.html
patching file webif/files/menu.html
patching file webif/files/menu_softcamkey.html
patching file webif/pages_index.txt
patching file webif/readerconfig/readerconfig_emubit.html`

**Comment by oscam-emu** [comment]
The patch you're using is outdated. You can clearly see that is not patching correctly in your logs...

**Comment by AnimaI** [comment]
ok^^
I used the current patch here ... I'll check it again tomorrow.

**Comment by oscam-emu** [comment]
I am closing this as invalid. If the issue is on our side and not your mistake, open a new issue at https://github.com/oscam-emu/oscam-patched/issues
